dev/com.ibm.ws.logstash.collector_fat/fat/src/com/ibm/ws/logstash/collector/tests/LogstashSSLTest.java

```diff
@@ -235,6 +235,7 @@ public void testLogstashForAuditEvent() throws Exception {
         testName = "testLogstashForAuditEvent";
         setConfig("server_logs_audit.xml");
         clearContainerOutput();
+        assertNotNull("The application is not ready", server.waitForStringInLogUsingMark("CWWKT0016I", 10000));
```

Comment:
This should be moved to the `setup()` method so that all tests will be run after the application is ready.